### PR TITLE
Add 7705

### PR DESCRIPTION
**Pull request by lmoelleb**
Datasheet: http://www.ti.com/lit/ds/symlink/lm7705.pdf

image

Capacitor pins set as input (I used 7660 as an example), not sure if passive would be a better fit. The datasheet refers to them as "analog".

------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required

### CODE REVIEW

**Comment by lmoelleb** [comment]
Would it make sense to stack the GND pins? I did not do it as the KLC contains these two statements:

> Power and ground pins should be set to either Power Input or Power Output

> Special Case: Pins of electrical type [Output, Power Output, Power Input] are special cases. Connecting these pins together causes ERC (electrical rule check) errors and must be avoided.

But then I randomly browsed through issues and found this:
https://github.com/KiCad/kicad-symbols/issues/1040
containing the statement from @evanshultz (emphasis mine):

> Our policy is to not stack power input (where the part draws power) pins if the datasheet calls out decoupling capacitors for each pin or gives other explicit reasons to keep the pins separate. _For ground pins or other pins which do not have explicit instructions about any per-pin requirements, we stack pins._

**Comment by DanSGiesbrecht** [comment]
GND pins should be stacked. The special case that you mention is illustrating that only **one** of the stacked group can be the correct type. That means that one of your two GND pins will be **visible, power input**, and the other will be **invisible, passive**. Having them both as **power inputs** would cause an ERC error in the stack. I usually go by lowest pin number as the dominant (visible) pin in a stack, unless there's an overriding reason (such as a pin being listed as the primary power pin).

File: Regulator_SwitchedCapacitor.dcm [hunk 1]
```diff
@@ -24,6 +24,12 @@ K Low-noise inverting charge pump with both positive and negative LDO's
 F http://www.ti.com/lit/ds/symlink/lm27762.pdf
 $ENDCMP
 #
+$CMP LM7705
+D Switched capacitor inverter, inverts 3V to 5.25V to regulated negative 0.23V, VSSOP-8
```
Comment (DanSGiesbrecht) [inline Regulator_SwitchedCapacitor.dcm]: Please let me know what you think of this description: `Negative bias generator, low-noise, 3-5.25V input, -0.232V fixed output, VSSOP-8`

File: Regulator_SwitchedCapacitor.dcm [hunk 2]
```diff
@@ -24,6 +24,12 @@ K Low-noise inverting charge pump with both positive and negative LDO's
...
+K switched capacitor voltage converter inverter negative bias regulator
```
Comment (DanSGiesbrecht) [inline Regulator_SwitchedCapacitor.dcm]: Along with the description suggestion above, here are keyword suggestions that are not present in the description: `switched cap capacitor converter inverter regulator supply`.

File: Regulator_SwitchedCapacitor.lib [hunk 3]
```diff
@@ -74,6 +74,29 @@ X C- 9 -400 -600 100 R 50 50 1 1 P
...
+DRAW
+S -300 300 300 -300 0 1 10 f
+X CF+ 1 -400 0 100 R 50 50 1 1 I
+X GND 2 0 -400 100 U 50 50 1 1 W
+X SD 3 -400 200 100 R 50 50 1 1 I
+X VDD 4 0 400 100 D 50 50 1 1 W
+X GND 5 0 -400 100 U 50 50 1 1 P N
+X VOUT 6 400 200 100 L 50 50 1 1 w
+X CRES 7 400 0 100 L 50 50 1 1 I
+X CF- 8 -400 -100 100 R 50 50 1 1 I
```
Comment (DanSGiesbrecht) [inline Regulator_SwitchedCapacitor.lib]: I think that you made a good suggestion: I think that `CRES`, `CF+` and `CF-` should be passive.

**Pull request merged**